FirebaseAdmin/FirebaseAdmin/Util/EnvironmentVariable.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace FirebaseAdmin.Util
+{
+    /// <summary>
+    /// Simplifies getting and setting  environment variables used in the project.
+    /// This class also serves as a dictionary, of sorts, for environment variables relevant to the project.
+    /// </summary>
+    internal static class EnvironmentVariable
+    {
+        /// <summary>
+        /// Gets or sets environment variable for connecting to the Firebase Authentication Emulator.
+        /// </summary>
+        /// <value>string in the form &lt;host&gt;:&lt;port&gt;, e.g. localhost:9099. Beware: No validation is done.</value>
+        internal static string FirebaseAuthEmulatorHost
```

Comment:
> let's move this constant into the new Utils class (and remove this class entirely), so it's scoped to the Auth namespace, and

I was actually thinking the class to also provide access to the "GOOGLE_CLOUD_HOST" and "GCLOUD_HOST" environment variables as well as other variables. I am quite unaware of the amount of env variables used for the Firebase client libraries, so I don't know if it's a good idea. However, it would make sense that this particular environment variable is scoped to the Auth namespace.

>(and remove this class entirely)

Removing the class itself, would result in naming like `FirebaseAuthHostEnvironmentVariable` instead of `EnvironmentVariable.FirebaseAuthHost`, which I see no advantage to. Am I misunderstanding you perhaps?

What do you think?

Reply:
Proposed change in https://github.com/firebase/firebase-admin-dotnet/pull/289/commits/85ee5259423d0ba2057c8f7854ae2c5348f5a104

Reply:
> Removing the class itself, would result in naming like FirebaseAuthHostEnvironmentVariable instead of EnvironmentVariable.FirebaseAuthHost, which I see no advantage to. Am I misunderstanding you perhaps?

My reasoning is that this class will end up being a wrapper for a single constant. Therefore it's just simpler to use the `Utils` class itself as the wrapper, and rename the constant to something like `EmulatorHostEnvironmentVar`.

Reply:
https://github.com/firebase/firebase-admin-dotnet/pull/289#discussion_r600870144